modules/cli/pages/index.adoc

```diff
@@ -51,12 +49,257 @@ $ mv skupper $HOME/bin
+
+. Generate a token on the cluster that you want to link to:
++
+--
+----
+$ skupper token create <filename> 
+----
+
+where `<filename>` is the name of a YAML file that is saved on your local filesystem.
+
+This file contains a key and the location of the site that created it.
+
+NOTE: Access to this file provides access to the {service-network}. Protect it appropriately.
+--
+
+. Use a token to join the {service-network}:
+
+.. Create a link:
++
+----
+$ skupper link create <filename> [-name <link-name>]
+----
++
+where `<filename>` is the name of a YAML file generated from the `skupper token create` command and `<link-name>` is the name of the link.
+
+.. Check the link:
++
+----
+$ skupper link status
+Connection for conn1 not active
+----
++
+In this example no <link-name> was specified, the name defaulted to `conn1`.
+
+
+. If you want to delete a link:
++
+----
+$ skupper link delete <link-name>
+----
+where `<link-name>` is the name of the link specified during creation.
+
+
+// Type: assembly
+[id="exposing-services"] 
+== Exposing services on the {service-network}
+
+After creating a {service-network}, exposed services can communicate across that network.
+
+The `skupper` CLI has two options for exposing services:
+
+* `expose` supports simple use cases, for example, a deployment with a single service.
+See xref:exposing-simple-services[] for instructions.
+
+* `service create` and `service bind` is a more flexible method of exposing services, for example, if you have multiple services for a deployment.
+See xref:exposing-complex-services[] for instructions.
+
+
+// Type: procedure
+[id="exposing-simple-services"] 
+=== Exposing simple services on the {service-network}
+This section describes how services can be enabled for a {service-network} for simple use cases.
+
+.Procedure
+
+. Create a deployment, some pods, or a service in one of your sites, for example to create the backend service for the link:{getting-started-link}[tutorial]:
++
+----
+$ kubectl create deployment hello-world-backend --image quay.io/skupper/hello-world-backend
+----
++
+This step is not {skupper-name}-specific, that is, this process is unchanged from standard processes for your cluster.
+
+. Create a service that can communicate on the {service-network}:
++
+--
+----
+$ skupper expose [deployment <name>|pods <selector>|statefulset <statefulsetname>|service <name>]
+----
+
+where 
+
+* `<name>` is the name of your deployment 
+* `<selector>` is a pod selector
+* `<statefulsetname>` is the name of a statefulset
+
+Options for this command include:
+
+`--port <port-number>`:: Specify the port number that this service is available on the {service-network}.
+
+`--target-port <port-number>`:: Specify the port number of pods that you want to expose.
+--
```

Comment:
Might be interesting to include the `--protocol` here as well.

Reply:
If we add --protocol, then we'll have to note that 'only tcp is supported', or something.. (nothing in dev preview is 'supported')
So, maybe add --protocol but upstream only?

Reply:
Sounds fair.